languages/en.ini

```diff
@@ -172,6 +172,8 @@ Catalog Results = "Catalog Results"
 catalog_login_desc = "Enter your library catalog credentials."
 CD = "CD"
 Change Password = "Change Password"
+channel_expand_icon = "Show more channels like this one"
+channel_search_icon = "Show channel as search results"
```

Comment:
The problem with "show as list" is that in the case of the "similar items" channels, there is no list. You just go to the record page.